protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -276,13 +278,25 @@ public function actionAdminUserSearch($keyword, $id)
 
     public function checkGroupAccess($group)
     {
-        if(!$group) {
+        if (!$group) {
             throw new HttpException(404, Yii::t('AdminModule.user', 'Group not found!'));
         }
 
-        if($group->is_admin_group && !Yii::$app->user->isAdmin()) {
+        if ($group->is_admin_group && !Yii::$app->user->isAdmin()) {
             throw new HttpException(403);
         }
     }
 
+    public function actionReassignAll($id)
```

Comment:
We should validate the `id` here.  Maybe load the `Group`.

Reply:
Done!